docker/nginx/libs/skynet/account.lua

```diff
@@ -62,16 +64,18 @@ end
 
 -- detect whether current user is authenticated
 function _M.is_authenticated()
+    if not _M.accounts_enabled() then return false end
+
     local limits = _M.get_account_limits()
 
-    return limits.tierName ~= anon_limits.tierName
+    return limits.tierID > tier_id_anonymous
```

Comment:
This doesn't play well with `local tier_id_internal = -1`. A direct comparison to `anonymous` might be better here.

Reply:
I removed internal tier so in light of that please reevaluate :)